TMessagesProj/src/main/java/org/telegram/messenger/MessagesStorage.java

```diff
@@ -4247,6 +4294,57 @@ public void run() {
             FileLog.e("tmessages", e);
         }
     }
+    private boolean isRequestedMe(TLRPC.Message message, int idMe, String nameMe)
+    {
+        if(message.to_id != null && message.to_id.user_id == message.dialog_id)
+        {
+            return false;
+        }
+        if(message.entities != null && message.entities.size() > 0)
+        {
+            for(TLRPC.MessageEntity e : message.entities)
+            {
+                if(e instanceof TLRPC.TL_messageEntityMentionName)
+                {
+                    if(((TLRPC.TL_messageEntityMentionName)e).user_id == idMe)
+                    {
+                        return true;
+                    }
+                }
+            }
+        }
+        if(message.fwd_from != null && message.fwd_from.from_id == idMe)
+        {
+            return true;
+        }
+        try
+        {
+            SQLiteCursor cursor = database.queryFinalized("SELECT data FROM messages WHERE mid = " + message.reply_to_msg_id);
```

Comment:
try find reply message in local database for compare from_id
this checking work bad and @DrKLO i need some help to understand how it works completely